tests/test_main.py

```diff
@@ -4,8 +4,135 @@
 from py21cmemu import Emulator
 
 
-def test_basic_prediction():
+def test_prediction():
     """Simply test that we can make a prediction without erroring."""
     emu = Emulator(version="latest")
     theta = np.random.rand(9 * 5).reshape((5, 9))
-    emu.predict(theta)
+
+    theta, output, errors = emu.predict(theta)
+
+    # Test writing
+    output.write("test_writing")
```

Comment:
Instead of writing to a file that will exist wherever you're running the tests from, use the pytest fixture `tmp_path` (https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-fixture) to write to a temporary location.